tests/cypress/tests/standalone/navigation_test.ts

```diff
@@ -100,6 +107,14 @@ Object.values(BasicModes).forEach(mode => {
                 DashTable.getCell(4, 2).should('have.class', 'focused');
                 DashTable.getCell(3, 1).should('not.have.class', 'focused');
             });
+
+            it('can select a cell outside of the viewport', () => {
```

Comment:
I think the test name could be improved. It's not about selecting a cell outside of viewport but rather about selecting a cell and scrolling it out of the viewport.

Like for the test above, would be interested in checking if there are console errors triggered by these steps. Maybe something like: https://github.com/cypress-io/cypress/issues/300#issuecomment-513514236

Reply:
@shammamah From your comment in https://github.com/plotly/dash-table/pull/535/files#r315342988, this is moot.